examples/complete/test.tfvars

```diff
@@ -10,19 +10,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-project_id = "<PROJECT_ID>"
-
-name = "demo-pipeline"
-
+project_id = "platform-accelerators"
+name       = "demo-pipeline"
 repository = {
-  repo_type             = "GitHub"
-  repo_id               = "launchbynttdata/demo-azure-resource-group"
-  github_enterprise_url = "https://github.company.com"
-  branch_name           = "main"
-  yml_path              = "internals/pipelines/pipeline.yml"
-  service_connection_id = "<SERVICE_CONNECTION_ID>"
+  repo_type   = "GitHub"
+  repo_id     = "launchbynttdata/tf-azureado-module_primitive-pipelines"
+  branch_name = "feature/support-github-account"
+  yml_path    = "examples/complete/pipeline.yml"
 }
 
+#Please provide the service connection name for the GitHub repository
+service_connection_name = "<Service Connection Name>"
+pull_request_trigger    = null
```

Comment:
I am curious how this works. There is a specific PR-handling construct in the ADO pipeline definition that looks almost identical to the `trigger` construct. See [documentation](https://learn.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#pr-triggers).

Reply:
If both `pipeline.yml` and any other trigger eg. `ci_trigger` are enabled, the `ci_trigger` configuration will take precedence over the `pipeline.yml` file. The `ci_trigger` block in the `azuredevops_build_definition` resource will be used to define the continuous integration triggers, overriding any triggers defined in the `pipeline.yml` file.
Used for overiding.

Reply:
Understood. Very good.